apps/src/applab/commands.js

```diff
@@ -1829,6 +1829,45 @@ var handleSetKeyValueSyncError = function(opts, message) {
   outputWarning(message);
 };
 
+applabCommands.getColumn = function(opts) {
+  apiValidateType(opts, 'getColumn', 'table', opts.table, 'string');
+  apiValidateType(opts, 'getColumn', 'column', opts.column, 'string');
+
+  Applab.storage.readRecords(
+    opts.table,
+    {},
+    handleGetColumn.bind(this, opts),
+    handleGetColumnError.bind(this, opts)
+  );
+};
+
+var handleGetColumn = function(opts, records) {
+  let columnList = [];
+  if (records === null) {
+    outputError(
+      'You tried to read records from a table called ' +
```

Comment:
Should these be i18n strings?

Reply:
esp since they are repeated in line 1904

Reply:
For some reason I had it in my head that we didn't support i18n error messages yet, but I think I was wrong. Fixed!